.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -238,6 +239,15 @@ jobs:
     displayName: "ORAS Push Artifacts in $(Build.ArtifactStagingDirectory)/linux/"
     condition: eq(variables.IS_PR, false)
 
+  - bash: |
+        set -euxo pipefail
+        oras attach ${{ variables.repoImageName }}:$(linuxImagetag) \
+            --artifact-type 'application/vnd.microsoft.artifact.lifecycle' \
+            --annotation "vnd.microsoft.artifact.lifecycle.end-of-life.date=$(date -u -d '-1 hour' +"%Y-%m-%dT%H:%M:%SZ")"
```

Comment:
1. why -1 hour, isnt that +1 hour? 
2.  after this timestamp, is image cant be pulled from MCR?
3. can we have this configurable through pipeline variable instead of hardcoding?

Reply:
1. Yes -1 means that it is already end of life as they are dev images
2. Yes it is just marked as end of life
3. it is doable but if the goal is to mark it EOL before push to mcr, I don't think it will require change, let me know your thoughts

Reply:
From #1 and #2, response,  dev images cant be used. is that right?

Reply:
yes it is not recommended to use by customers and it won't be scanned for vulnerabilities